R/list_as_json_char.R

```diff
@@ -0,0 +1,24 @@
+#' Helper function to convert R list to JSON-like string
```

Comment:
I think it can be deleted. seems that we don't need to convert r lists to son like strings

Reply:
I removed it

Reply:
Yes, agreed. It might depend on our future success (hopefully, there is any) on the asset creation, where we might have to provide the `settings` value as a json string... But I agree, we can add potentially needed helper functions in the future.